applications/ShapeOptimizationApplication/tests/sliding_opt_test/optimization_parameters.json

```diff
@@ -0,0 +1,70 @@
+{
+    "optimization_settings" : {
+        "model_settings" : {
+            "domain_size"           : 3,
+            "model_part_name"       : "3D_Shell",
+            "model_import_settings" : {
+                "input_type"        : "mdpa",
+                "input_filename"    : "3D_shell_51n"
+            },
+            "design_surface_sub_model_part_name" : "design_surface",
+            "damping" : {},
+            "mesh_motion" : {
+                "apply_mesh_solver" : false
+            }
+        },
+        "objectives" : [{
+            "identifier" : "reciprocal_r_squared_sum",
+            "type"       : "minimization",
+            "analyzer"   : "external",
+            "project_gradient_on_surface_normals" : false
+        }],
+        "constraints" : [],
+        "design_variables" : {
+            "type"   : "vertex_morphing",
+            "filter" : {
+                "filter_function_type"       : "linear",
+                "filter_radius"              : 25.0,
+                "max_nodes_in_filter_radius" : 10000,
+                "sliding_morphing" : true,
+                "sliding_morphing_settings" : {
+                    "model_import_settings" : {
+                        "input_type"     : "mdpa",
+                        "input_filename" : "boundary_surface_35n"
+                    },
+                    "background_sub_model_part_name" : "boundary",
+                    "sliding_sub_model_part_name" : "sliding_edge"
+                }
+            }
+        },
+        "optimization_algorithm" : {
+            "name"               : "steepest_descent",
+            "max_iterations"     : 2,
```

Comment:
Would one step be enough? In that case you would need to compare the `SHAPE_UPDATE` result.